client/src/components/CMenu.style.js

```diff
@@ -92,11 +91,10 @@ export const MenuElement = styled.button`
   border: none;
   padding: 0.5rem;
   text-align: center;
-  -webkit-box-shadow: 2px 2px 5px 0px rgba(0, 0, 0, 0.75);
-  -moz-box-shadow: 2px 2px 5px 0px rgba(0, 0, 0, 0.75);
-  box-shadow: 2px 2px 5px 0px rgba(0, 0, 0, 0.75);
+  -webkit-box-shadow: 2rem 2rem 5remx 0rem rgba(0, 0, 0, 0.75);
+  -moz-box-shadow: 2rem 2rem 5rem 0rem rgba(0, 0, 0, 0.75);
+  box-shadow: 2rem 2rem 5rem 0rem rgba(0, 0, 0, 0.75);
```

Comment:
I think all of these can be px since they're just box shadow! Changing from px to rem like this makes them huge 😆

Reply:
Did you mean to change `rem ` to `px ` . @Oliversw